nautobot_ssot/integrations/slurpit/jobs.py

```diff
@@ -0,0 +1,144 @@
+"""Slurpit DataSource job class."""
+
+import slurpit
+from django.contrib.contenttypes.models import ContentType
+from django.templatetags.static import static
+from django.urls import reverse
+from nautobot.dcim.models import Device, LocationType
+from nautobot.extras.choices import SecretsGroupAccessTypeChoices, SecretsGroupSecretTypeChoices
+from nautobot.extras.jobs import BooleanVar, Job, ObjectVar
+from nautobot.extras.models import ExternalIntegration
+from nautobot.ipam.models import Namespace
+
+from nautobot_ssot.integrations.slurpit import constants
+from nautobot_ssot.integrations.slurpit.diffsync.adapters.nautobot import NautobotDiffSyncAdapter
+from nautobot_ssot.integrations.slurpit.diffsync.adapters.slurpit import SlurpitAdapter
+from nautobot_ssot.jobs.base import DataMapping, DataSource
+
+
+class SlurpitDataSource(DataSource, Job):  # pylint: disable=too-many-instance-attributes
+    """SSoT Job class."""
+
+    credentials = ObjectVar(
+        model=ExternalIntegration,
+        queryset=ExternalIntegration.objects.all(),
+        display_field="name",
+        required=True,
+        label="Slurpit Instance",
+    )
+
+    building_loctype = ObjectVar( # pylint: disable=R0801
+        model=LocationType,
+        queryset=LocationType.objects.all(),
+        display_field="name",
+        required=False,
+        label="Building LocationType",
```

Comment:
This should be changed to Site LocationType if Slurpit calls locations Sites.

Reply:
This has been updated now.

Reply:
Confirmed.